tests/affiliatedcertification/affiliatedcerthelper/affiliatedcerthelper.go

```diff
@@ -33,6 +33,8 @@ func SetUpAndRunContainerCertTest(containersInfo []string, expectedResult string
 
 	err = globalhelper.LaunchTests(
 		[]string{affiliatedcertparameters.AffiliatedCertificationTestSuiteName},
+		affiliatedcertparameters.TestCaseContainerAffiliatedCertName,
```

Comment:
agree,
@theemj  Hey Emily, please let me know how can I make the folder name shorter

Reply:
hmm we could just use "container-is-certified" and "operator-is-certified", we'd just need to add two parameters to affiliatedcertparameters like:

```
ContainerTestCaseName = "container-is-certified"
OperatorTestCaseName =  "operator-is-certified"
```

Reply:
@theemj  Let's talk about it please, because only one file is being generated for all of your tests, and it keeps getting deleted and written again every new launch, thus every test needs a different name.

Reply:
Ah I see. I could change the SetUpAndRunContainerCertTest parameters so that a unique name is passed in for each test case

Reply:
@theemj Hey, attached a screenshot showing the tree of the file inside affiliated folder, please confirm that this is ok.
also, 3 tests are missing because they have failed before running the tnf, so no log file is generated for them, the log file contains only tnf logs, not QE logs.

![image](https://user-images.githubusercontent.com/51234399/160572018-f59ae139-de1b-47cb-9a8a-5225ac5baa6f.png)

Reply:
@greyerof Yes the name of the files are long but on the other hand it's easy to distinguish between test cases. We can not use polarion tags because it's our internal tool.

Reply:
I'm ok if it works for you, guys.
I've worked with both methods (numbers and long names) in the past, and just as per my experience, it was actually harder navigating through folders with so long file names inside. It made copy/moving operations more difficult and I ended up renaming the failing TC ones to shorter names when I needed to open them for review.
At the end of the day, the normal scenario is that you'll get only one or two failing TCs/regressions per QE run due someone adding wrong code in the tnf repo. Searching for "23433.log" and "55043.log" files (both example tc numbers) and their code in your repo based with a grep of those numbers seems easier/faster than dealing with those long names.
Also, you don't really need to use polarion numbers. You can start a new numbering scheme independent from polarion's one. For example, first QE TS (networking?) can start with 101, 102, 103... second test suite (lifecycle?) can start with 201, 202, 203... I understand you may not want supporting two different numberings schemes, though.
Anyway, as I said, that's all based on my personal experience, I've never worked on QE, so I only see these things from the developer's side.